README.md

```diff
@@ -155,9 +159,11 @@ You will now edit the `mcpServers` section of the configuration file.
         "run",
         "--rm",
         "crystaldba/postgres-mcp",
-        "postgresql://username:password@localhost:5432/dbname",
         "--access-mode=unrestricted"
-      ]
+      ],
```

Comment:
Docker needs a `-e` argument to pass in the environment variable.

Reply:
Done.